lib/puppet_x/puppetlabs/aws.rb

```diff
@@ -162,6 +162,14 @@ def elb_client(region = default_region)
         self.class.elb_client(region)
       end
 
+      def self.elbv2_client(region = default_region)
```

Comment:
@MWilkinson Looking forward to those new types.  I think I should be able to help test those.